tasks/main.yml

```diff
@@ -35,15 +35,6 @@
   changed_when:
     - dep_install.stdout is regex("[1-9]\d* upgraded|[1-9]\d* newly installed")
 
-- name: Upgrade all packages to the latest version
-  apt:
```

Comment:
Can make it OS independent with https://docs.ansible.com/ansible/latest/modules/package_module.html